lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -414,6 +414,7 @@ sub _prepare_request_data {
     my $msg_type = $req_storage->{msg_type} ||= $req_storage->{method};
 
     my $params       = $self->make_call_params($c, $req_storage);
+    $params->{correlation_id} = $c->stash('correlation_id');
```

Comment:
Isn't it already included in `$req_storage->{stash_params}`?

Reply:
this will be changed already discussed with Leonid and Mehdi this was getting lost when new request popping in where as stash is maintained for a mojo session as we are already passing multiple params in stash